docs/io/visualization/how_to_liv_plot.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>We should remove gridlines by default -- they get very cluttered on the log plots. If the user wants them, they're easy to add</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>